.gemini/styleguide.md

```
@@ -0,0 +1,178 @@
+| 🔴 CRITICAL | ドメイン層・インフラ層で `anyhow` を直接使っている（`thiserror` を使うべき） | エラーハンドリング規約 |
+| 🔴 CRITICAL | アプリ層がインフラ層の具体型に直接依存している（Trait経由でない） | DI原則 |
+| 🔴 CRITICAL | ロジック変更にテストが伴っていない | テスト戦略 |
+| 🔴 CRITICAL | `engine/systems/` 以外(cli等)でゲームルールロジックを実装している | engineクレート構成規約 |
+| 🟡 HIGH | 日本語コメントがない、または意図がわからないコメントのみ | コードスタイル規約 |
+| 🟡 HIGH | 不必要な `.clone()` がある | メモリライフサイクル規約 |
+| 🟡 HIGH | `Box<dyn Trait>` のライフサイクルが不明瞭 | メモリライフサイクル規約 |
+| 🟡 HIGH | SOLID原則違反（単一責任・開放閉鎖など） | SOLID/DRY原則 |
+| 🟡 HIGH | DRY違反（重複ロジック） | SOLID/DRY原則 |
+| 🟡 HIGH | `engine/components/` に描画用データが混入している | engineクレート構成規約 |
+| 🟡 HIGH | プリミティブ型（`i32`, `String` 等）を直接フィールドに使用している | Newtypeパターンの徹底 |
+重要な問題を特定し、解決して全体的なコード品質を向上させることを目指してください。細かい問題は意図的に無視してください。
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)